test/cpp/BUILD

```diff
@@ -137,6 +142,7 @@ ptxla_cc_test(
             ":torch_xla_test",
             "//torch_xla/csrc/runtime:metrics",
             "//torch_xla/csrc:tensor",
+            "//torch_xla/csrc:aten_cuda_functions",
```

Comment:
I'm using the fallback implementation for solving the undefined references in C++ tests. I think this should be reasonable, since we don't test fallback on C++ tests.